src/components/components/join-cta/join-cta.js

```diff
@@ -20,7 +20,12 @@ const JoinCta = props => {
     props.className
   }`;
   return (
-    <a href={`${BACKEND_URL}/oidc/login`} className={classString}>
+    <a
+      href={`${BACKEND_URL}/oidc/login`}
+      target="_blank"
+      rel="noopener noreferrer"
+      className={classString}
```

Comment:
this link is for the login so we shouldn't include them here

Reply:
okay